l5kit/l5kit/geometry/transform.py

```diff
@@ -33,55 +33,6 @@ def yaw_as_rotation33(yaw: float) -> np.ndarray:
     return transforms3d.euler.euler2mat(0, 0, yaw)
 
 
-def world_to_image_pixels_matrix(
```

Comment:
Please can we have this back with the new code to compute it? You have 6 lines of code repeated in 4 places to compute this. It's valuable to have a function to do it

Reply:
```python
def raster_from_world(render_context: RenderContext, ego_translation_m: np.ndarray, ego_yaw_rad: float) -> np.ndarray:
    """
    Return a matrix to convert a world pose into raster coordinates

    Args:
        render_context (RenderContext): the context for rasterisation
        ego_translation_m (np.ndarray): XY translation in world coordinates
        ego_yaw_rad (float): yaw angle

    Returns:
        (np.ndarray): 3x3 matrix
    """
    # Compute ego pose from its position and heading
    ego_pose = np.array([[np.cos(ego_yaw_rad), -np.sin(ego_yaw_rad), ego_translation_m[0]],
                         [np.sin(ego_yaw_rad), np.cos(ego_yaw_rad), ego_translation_m[1]],
                         [0, 0, 1]])

    ego_from_world = np.linalg.inv(ego_pose)
    return render_context.raster_from_local @ ego_from_world
```

Reply:
make sense

Reply:
I moved `raster_from_world` function to render context class